scripts/install_third_party.sh

```diff
@@ -27,13 +27,10 @@ install_node_module eslint 4.19.0
 install_node_module eslint-plugin-angular 0.12.0
 install_node_module eslint-plugin-html 4.0.1
 install_node_module gulp 3.9.0
-install_node_module gulp-clean-css 2.0.2
-install_node_module gulp-concat 2.6.0
-install_node_module gulp-sourcemaps 1.6.0
-install_node_module gulp-uglify 2.0.1
-install_node_module gulp-util 3.0.7
 install_node_module through2 2.0.0
 install_node_module yargs 3.29.0
+install_node_module gulp-concat 2.6.0
```

Comment:
Might as well keep the existing alphabetical ordering?

Reply:
Done.